AirBench/Repository/BenchApiRepository.cs

```diff
@@ -27,20 +27,68 @@ public BenchApiRepository (Context context)
         //    return myBench;
         //}
 
-        async public Task<List<BenchInfo>> GetBenchList()
+        async public Task<List<BenchList>> GetBenchList()
         {
-            List<Bench> benches = await context.Benches.ToListAsync();
-            List<BenchInfo> benchResponses = new List<BenchInfo>();
-            foreach(var bench in benches)
+            ReviewRepository repo = new ReviewRepository(context);
+            LoginRepository Repository = new LoginRepository(context);
+            List<Bench> myBenches = await context.Benches.ToListAsync();
+            List<BenchList> myBenchList = new List<BenchList>();
+            foreach (var bench in myBenches)
             {
-                BenchInfo myResponse = new BenchInfo();
-                myResponse.Description = bench.Description;
-                myResponse.Latitude = bench.Latitude;
-                myResponse.Longitude = bench.Longitude;
-                myResponse.NumberOfSeats = bench.NumberOfSeats;
-                benchResponses.Add(myResponse);
+                User user = Repository.GetById(bench.CreatorUserId);
+                var name = user.Name.Split(' ');
+                var reviews = repo.GetReviewList(bench.Id);
+                BenchList currentBench = new BenchList();
+                currentBench.CreatorUserId = bench.CreatorUserId;
+                if (!(name.Count() > 1))
+                {
+                    currentBench.Name = name[0];
+                }
+                else
+                {
+                    currentBench.Name = name[0] + ' ' + name[1].ToCharArray()[0] + '.';
+                }
```

Comment:
This is the second time I've been this code to produce the first and last name initial string. Seems like a great opportunity for a `User` model class getter-only property.